content/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes.md

```diff
@@ -0,0 +1,144 @@
+
+* **Ability to control how bucket timestamps are reported**
+
+  Till version 1.8, [TS.RANGE](https://redis.io/commands/ts.range/), [TS.REVRANGE](https://redis.io/commands/ts.revrange/), [TS.MRANGE](https://redis.io/commands/ts.mrange/), and [TS.MREVRANGE](https://redis.io/commands/ts.mrevrange/) returned the start time of each reported bucket as its timestamp.
+
+  Changing this behavior and reporting each bucket’s start time, end time, or mid-time is now possible. This is required in many use cases. For example, when drawing bars in trading applications, annotating each bar with the end timestamp of the bucket it represents is very common.
+
+  The way bucket timestamps are reported can be specified with the new optional `BUCKETTIMESTAMP` parameter of [TS.RANGE](https://redis.io/commands/ts.range/), [TS.REVRANGE](https://redis.io/commands/ts.revrange/), [TS.MRANGE](https://redis.io/commands/ts.mrange/), and [TS.MREVRANGE](https://redis.io/commands/ts.mrevrange/):
+
+  `-` or `low`: the bucket's start time (default)
+
+  `+` or `high`: the bucket's end time
+
+  `~` or `mid`: the bucket's mid-time (rounded down if not an integer)
+
+* **Ability to control alignment for compaction rules**
+
+  Till version 1.8, compaction rules could not be aligned. One could specify a compaction rule with 24-hour buckets, and as a result, each bucket would aggregate events from midnight till the next midnight. The first bucket always started at the epoch and all other buckets were aligned accordingly.
+
+  But what if we want to aggregate daily events from 06:00 to 06:00 the next day? We can now specify alignment for compaction rules.
+
+  Alignment can be specified with the new optional `alignTimestamp` parameter of [TS.CREATERULE](https://redis.io/commands/ts.createrule/) and the [COMPACTION_POLICY](https://redis.io/docs/stack/timeseries/configuration/#compaction_policy) configuration parameter. Specifying `alignTimestamp` ensures that there is a bucket that starts exactly at `alignTimestamp` and all other buckets are aligned accordingly. `alignTimestamp` is expressed in milliseconds. The default value is 0 (aligned with the epoch).
+
+* **New reducers**
+
+  Till version 1.8, only the `sum`, `min`, and `max` could be used as reducer types.
+
+  It is now possible, for example, to calculate the maximal temperature per timeframe for each sensor and then report the average temperature (avg reducer) over groups of sensors (grouped by a given label's value).
+
+  This can be specified with the new reducer types ([TS.MRANGE](https://redis.io/commands/ts.mrange/) and [TS.MREVRANGE](https://redis.io/commands/ts.mrevrange/)): `avg`, `range`, `count`, `std.p`, `std.s`, `var.p`, and `var.s`.
+
+{{<note>}}
+RDB version 7 since 1.8.2. RDB files are not backward compatible.
```

Comment:
Is there a missing word in the first sentence?

Reply:
Fixed. Now similar to what we have in v1.6.16